primo/data_parser/default_data.py

```diff
@@ -297,33 +297,45 @@ class _SupportedContent:
     ),
 }
 
+# note the names of these metrics must be identical to
+# a property of the Optimal Project class for the
+# efficiency calculation
 SUPP_EFF_METRICS = {
     "num_wells": _SupportedContent(
-        name="num_wells",
-        full_name="Number of Wells",
+        name="num_wells", full_name="Number of Wells", required_data="well_id"
     ),
     "num_unique_owners": _SupportedContent(
         name="num_unique_owners",
         full_name="Number of Unique Owners",
+        required_data="operator_name",
+        has_inverse_priority=True,
     ),
-    "dist_centroid": _SupportedContent(
-        name="dist_centroid",
-        full_name="Distance to Centroid [miles]",
-    ),
-    "elevation_delta": _SupportedContent(
-        name="elevation_delta",
+    "avg_elevation_delta": _SupportedContent(
+        name="avg_elevation_delta",
         full_name="Average Elevation Delta [m]",
+        required_data="elevation_delta",
+        has_inverse_priority=True,
     ),
     "age_range": _SupportedContent(
         name="age_range",
         full_name="Age Range [Years]",
+        required_data="age",
+        has_inverse_priority=True,
     ),
     "depth_range": _SupportedContent(
         name="depth_range",
         full_name="Depth Range [ft]",
+        required_data="depth",
+        has_inverse_priority=True,
     ),
     "record_completeness": _SupportedContent(
         name="record_completeness",
         full_name="Record Completeness",
     ),
+    "avg_dist_to_road": _SupportedContent(
```

Comment:
Same as above. Change it back to `dist_to_road` and add `fill_missing_value` attribute
fill_missing_value={"domain": NonNegativeFloat, "default": 0},

Reply:
Added.

Reply:
Name change in a future PR